README.md

```diff
@@ -742,6 +742,33 @@ You will find information on the repair operation in the `LOG` file inside the d
 
 Before calling `repair()`, close a database if it's using the same `location`.
 
+## Ordering
+
+Write operations don't have a defined order because they are asynchronously dispatched on the Node.js thread pool.
```

Comment:
```suggestion
Write operations don't have a defined order because they are asynchronously dispatched.
```

Consumers of this module shouldn't need to know about the thread pool